eng/common/TestResources/TestResources-Helpers.ps1

```diff
@@ -149,6 +149,33 @@ function BuildBicepFile([System.IO.FileSystemInfo] $file) {
     return $templateFilePath
 }
 
+function LintBicepFile([string] $path) {
+    if (!(Get-Command bicep -ErrorAction Ignore)) {
```

Comment:
We need to reconcile this with https://github.com/Azure/azure-sdk-tools/pull/10550 cc @hallipr